server/controllers/UserEventsActivitiesController.js

```diff
@@ -5,48 +5,51 @@ const {
   filterEventData,
 } = require("../services/UserEventsActivitiesService");
 
+const {
+  ConflictError,
+  UnauthorizedError,
+  BadRequestError,
+  NotFoundError,
+} = require("../utils/errors");
 //@endpoint GET /userActivities
 //@desc get all events that the user joined
 //@access PRIVATE - will need to validate token? YES
 const handleUserActivities = async (req, res, next) => {
   try {
     let userId = req.params.userId;
     if (!userId) {
-      res.status(400).json({
-        status: 400,
-        message: "Missing user credentials in handleUserActivities.",
-      });
+      let err = new UnauthorizedError(
```

Comment:
Missing a userId should be a `BadRequest` not `Unauthorized`. We don't know if the user is authorized or not at this point